hubverse_annotator/ui.py

```diff
@@ -31,7 +31,13 @@
 CHART_TITLE_FONT_SIZE = 18
 REF_DATE_STROKE_WIDTH = 2.5
 REF_DATE_STROKE_DASH = [6, 6]
+MARKER_SIZE = 65
 ROOT = pathlib.Path(__file__).resolve().parent.parent
+CI_SPECS: dict[str, dict[str, object]] = {
+    "97.5% CI": {"low": "0.025", "high": "0.975", "color": "#DEEBF7"},
```

Comment:
```suggestion
    "95% CI": {"low": "0.025", "high": "0.975", "color": "#DEEBF7"},
```

Might be nice to derive this programmatically, to avoid this mistake.

Reply:
I was referring to the incorrect "97.5%" label